lib/features/receipt/presentation/widgets/list_entry/purchase_receipt_list_entry.dart

```diff
@@ -10,18 +11,32 @@ class PurchaseReceiptListEntry extends StatelessWidget {
     required this.receipt,
   });
 
+  String get priceText {
+    final price = receipt.price;
+    return switch (receipt.paymentStatus) {
+      PaymentStatus.completed => '$price,-',
+      PaymentStatus.awaitingPayment ||
+      PaymentStatus.reserved ||
+      PaymentStatus.refunded =>
+        '($price,-)',
+      PaymentStatus.error || PaymentStatus.rejectedPayment => '',
+    };
+  }
+
   @override
   Widget build(BuildContext context) {
     return ReceiptListEntry(
-      tappable: true,
+      tappable: false,
```

Comment:
how come we no longer want to be able to open a purchase? this removes users ability to view the exact timestamp of the purchase that is displayed in the opened receipt

Reply:
#477. The "receipt" of a purchase (especially incomplete purchases) should never resemble like a swipe receipt.

At some point we should add the purchase receipt page as specified in the v4 design spec (screenshot below)
![image](https://github.com/AnalogIO/coffeecard_app/assets/21163286/87b2dfbe-2e2b-4c18-a6ba-146533f9d267)

Reply:
i agree, but as it stands it now removes users ability to view the exact timestamp of the purchase. Might i suggest we disable this feature when migrating to the new design?

Reply:
Then we'll just add the timestamp to the receipt list entry for now. See latest commit